phpBB/language/en/acp/styles.php

```diff
@@ -80,6 +80,7 @@
 	'STYLE_UNINSTALL'			=> 'Uninstall',
 	'STYLE_UNINSTALL_DEPENDENT'	=> 'Style "%s" cannot be uninstalled because it has one or more child styles.',
 	'STYLE_UNINSTALLED'			=> 'Style "%s" uninstalled successfully.',
+	'STYLE_PHPBB_VERSION'		=> 'Style phpBB version',
```

Comment:
I would prefer an different wording and an explanation.
"Style for phpBB version" or similar.

Reply:
Just want to point out that the label for the Extension details is _phpBB Version_ and is under a _Requirements_ section:
![image](https://user-images.githubusercontent.com/186341/35222709-c9a6e1dc-ff4c-11e7-8db6-b6fe65aae3b0.png)

We should probably be consistent with styles, plus this affords styles authors the ability to add other requirements details that can be displayed in the future (even though they wouldn't be displayed right now, but they could be).